app/views/messages/index.html.erb

```diff
@@ -8,7 +8,12 @@
   <h2>Contact History</h2>
   <% if @contact_history.present? %>
     <ul class="contact-history">
+      <% last_date = nil %>
       <% @contact_history.each do |contact_record| %>
+        <% if contact_record.datetime.to_date != last_date %>
+          <li class="contact-record__day-heading"><%= if contact_record.datetime.to_date == Date.today then "Today" end %> <%= contact_record.datetime.to_date.strftime("%m/%d/%Y") %></li>
+          <% last_date = contact_record.datetime.to_date %>
+        <% end %>
```

Comment:
Realized that we may have to change this when we start displaying things in the user's timezone, but this makes sense since we're only displaying UTC right now.

Reply:
Shannon pairing with Ben + Sarah:

Have you considered group_by to group the records by date before heading into the view?

@records_by_day = @contact_history.group_by { |record| record.created_at.beginning_of_day }

{key is the datetime beginning of day => []}

Reply:
We considered it, and we chose this approach for simplicity. Asheesh has a preference for flatter data structures that comes out of his time writing Python (they have a motto in Python land about his apparently, "flat is better than nested"). I think either approach works, so went with his preference.